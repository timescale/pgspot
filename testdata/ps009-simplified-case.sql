SELECT
    CASE a OPERATOR(pg_catalog.=) b
        WHEN true THEN 'true'
        WHEN false THEN 'false'
    END
    FROM my_schema.foo;
