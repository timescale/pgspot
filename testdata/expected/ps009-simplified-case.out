PS009: Unsafe CASE expression: CASE a OPERATOR(pg_catalog.=) b WHEN TRUE THEN 'true' WHEN FALSE THEN 'false' END at line 1

 Errors: 1 Warnings: 0 Unknown: 0 

